src/WebJobs.Script/Workers/SharedMemoryDataTransfer/MemoryMappedFileAccessorUnix.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.SharedMemoryDataTransfer
     /// </summary>
     public class MemoryMappedFileAccessorUnix : MemoryMappedFileAccessor
     {
-        private IEnvironment _environment;
+        private readonly IEnvironment _environment;
```

Comment:
Any reason for making this `readonly`? If so, can you also check if we need to do the same in `MemoryMappedFileAccessorWindows.cs`?

Reply:
I believe this is to prevent unintentionally setting any environmental variables, as the class should be using the `_environment` variable only to get a list of directories where it can create a memory-mapped file. Currently, no methods set environmental variables, but the `readonly` modifier may nonetheless be a useful safeguard for future modifications.

Reply:
As for the `MemoryMappedFileAccessorWindows.cs`, we have no corresponding `_environment` variable because the `MemoryMappedFile.CreateNew` method by default creates a memory-mapped file that is not mapped to an existing file on the disk (see [here](https://docs.microsoft.com/en-us/dotnet/standard/io/memory-mapped-files)).